recipes/tts/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

No need for such a high requirement within this CMake file

Reply:
Done